### PR TITLE
Microverse Module One: Capstone Project

### PR DESCRIPTION
**Pull request by DannieMo**
This project contains the HTML, CSS, and JavaScript Capstone Project. Capstone Projects are done at the end of a module at Microverse, a remote tech institution.

This project is a two pages website built for an annual global conference: CEOs International Community Global Summit.

Read through my readme to get the full details, see images and even download the repo for this conference page website. 

Thank😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,67 @@
...
+
+1. Set up:
+   Download this Github Repository https://github.com/DannieMo/MicroverseModule1-Capstone-Project.git and go to the index.html file
+
+2. Install:
+   Set up liveserver as an extention in your VS Code
+
+3. Usage:
+   Right click and open the index.html in your browser to view
+
```
Comment (mimipeshy) [inline README.md]: - [ ] Kindly submit a video presentation that contains a demo of the project, showing the pages in different screen sizes. Also, one thing you are proud of 😄 
- [ ] Kindly make use of  GitHub Pages to deploy the web pages. Ensure that the project is deployed properly, all pages, links, and images work as expected. Lastly, the link to the deployed project should be present in the README.

File: html/about.html [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        <img src="/assets/about7.jpg" alt="" />
+      </div>
+    </section>
+
+    <footer>
+      <div class="footer-flex">
+        <div class="footer-logo">
+          <a href="index.html"
+            ><img src="/assets/cic_logo_Real.png" alt="logo"
+          /></a>
```
Comment (mimipeshy) [inline html/about.html]: - [ ] The logo link in the footer section is broken, kindly fix it

File: html/index.html [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="webpages">
+              <a href="#">News</a>
+            </li>
+          </ul>
+
+          <div class="company-name">
+            <a href="index.html">CIC Campaign</a>
+          </div>
```
Comment (mimipeshy) [inline html/index.html:76]: - [ ] The navbar items should be in a single line. The CIC campaign overlaps to the next line displacing news nav item ⏬ 
ci
Kindly work on this

File: html/index.html [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+        <div class="program-list">
+          <div class="lecture flex-box">
+            <img class="icons" src="/assets/program1.png" alt="program-icon" />
+            <h2>Lecture</h2>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Deserunt,
+              veritatis!.
+            </p>
+          </div>
+
```
Comment (mimipeshy) [inline html/index.html:121]: - [ ] The whole section in small mobile views of up to 550px, the information is squeezed. Kindly add margins and fix the height of the images ⬇️ 

bn

- [ ] From html semantics, the whole section should be wrapped in an `article` tag

File: html/index.html [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+          <a href="index.html"
+            ><img src="/assets/cic_logoReal.png" alt="footer_logo"
+          /></a>
+        </div>
+        <div class="copyright">
+          <p class="copyright">
+            2021 CEOs Community Nigeria
+            <br />
+            SomeRights Reserved
+          </p>
```
Comment (mimipeshy) [inline html/index.html:214]: - [ ] the footer in small mobile view overlaps to the container above. Kindly fix this ⬇️ 

kl

File: html/about.html [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+        <h2>See The Past CEOs International Community Global Summits</h2>
+        <hr />
+        <p>
+          Take a look at the last two CIC Global Summits which took place in
+          Sounth Africa and in Abuja, Nigeria.
+        </p>
+        <img src="/assets/about4.jpg" alt="" />
+        <img src="/assets/about7.jpg" alt="" />
+      </div>
+    </section>
```
Comment (mimipeshy) [inline html/about.html]: - [ ] In this section on the desktop view the two bottom columns should be centered, in a horizontal line and have a gradient over them when a user hovers over the image in both mobile and desktop view. 

Clone | Original
------------ | -------------
jh | km

File: html/about.html [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+          The logo of CIC Global Summit 2021 was decided through the logo
+          competition from 8 June to 7 July. After online public voting and
+          Selection Committee's judging process, the logo from Naresh Agrawal
+          won the prize. After consultation with Mr. Agrawal we optimized the
+          prize-winning-logo for this year's summit as follows:
+        </p>
+      </div>
+      <div class="logo-container">
+        <img src="/assets/cic_logoReal1.png" alt="logo" />
+      </div>
```
Comment (mimipeshy) [inline html/about.html:133]: - [ ] The image is overflowing outside of the container in mobile view making a horizontal line in mobile view. Kindly fix this
hj

File: html/index.html [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+            Join CIC Global Summit 2021
+          </button>
+        </div>
+      </div>
+    </section>
+    <div class="section-speakers-heading">
+      <h2>Featured Speakers</h2>
+      <hr />
+    </div>
+
```
Comment (mimipeshy) [inline html/index.html:161]: - [ ] From html semantics, the whole section should be wrapped in an `article` tag

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @DannieMo  👋🏾 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
#### Highlights

- good job on the ui but lets make it better✔️ 
- good GitHub workflow ✔️ 
- not forming any linter errors ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @DannieMo** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- Kindly As the Previous TSE mentioned, you need to upload a video for the project where e you should present the project + a working live demo link.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DannieMo** [comment]
Hello sir, i have been done with my project hours ago but my network has
been very poor. I tried hosting my app using Netlify and Heroku but the
issues still persist because of my poor network. The network issue is from
my service providers and there's nothing much I can do from my  ends.

 It's almost midnight now: deadline is just a few minutes away. Pls help me
move on to the next phase of my Microverse journey.

Thanks.

DannieMo

**Comment by DannieMo** [comment]
Still giving same response: page not found.

Pls help me.

Thanks

Dannie

**Comment by DannieMo** [comment]
I just sent the loom presentation video now. Pls check it and let me know
what you think.

I really want to move on to the next phase of Microverse. Pls help me.

Thanks.

Dannie

**Review by dasileker — APPROVED** [review]
### **Hi @DannieMo** 

### **Project Approved ✅✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢
Congratulations! 🎉 

### **Optional**

- For the  Live demo link you can use the [rawgithag.com](https://raw.githack.com/) by selecting the html file link  like yours for example: [https://github.com/DannieMo/MicroverseModule1-Capstone-Project/blob/features/html/index.html] and paste it then take the dev link or the production link. now you have your lie demo

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**